### DISCUSSION TITLE
how can i do this

### DISCUSSION THREAD

**Discussion by abuhabban** [termux/termux-app#3606] [General]
how can i make my terminal like this. i have circled them with red colour.
![188518018-37add37b-2ab8-4b8e-9194-866e915cc4e0](https://github.com/termux/termux-app/assets/70461458/e3c75424-62cb-4439-b602-97c0b8c44c94)
i wantbto do same thing in my termux because it show completing time and directory in cool way etc.

**Comment by TomJo2000** [comment]
What you have circled are the "right prompts" supported on Shells like Zsh, Fish or Nushell.
### **Bash**(the default Shell) **does not support these.**
The specifics of how you setup and configure them are gonna vary for each Shell.
If you would like a Shell agnostic prompt configuration utility, we have [starship](https://starship.rs/presets/) packaged in the repositories,
so you can just install it with `pkg i starship`.
Additional information on right prompts is also provided in the [starship documentation](https://starship.rs/advanced-config/#enable-right-prompt).

**Comment by eamonburns** [comment]
I believe what you are looking for is [`zsh`](https://github.com/ohmyzsh/ohmyzsh/wiki/Installing-ZSH) with the [`powerlevel10k`](https://github.com/romkatv/powerlevel10k) theme.

That theme gives a lot of configuration options, and you will probably be able to get your prompt to look exactly like what is on the image.
